meta-zarhus-webkit/recipes-networking/cog/cog_%.bbappend

```
@@ -0,0 +1,31 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+DEPENDS:append:${PN}-autostart = " \
```

```suggestion
RDEPENDS:append:${PN}-autostart = " \
```

`DEPENDS` means it only needs it during build. And does it really need to depend on nginx? You could add variable with default starting URI. You could probably even run html file without server.

---

> And does it really need to depend on nginx? You could add variable with default starting URI. You could probably even run html file without server.

It does. It is a demo, that uses nginx and systemd service.

---

I'm afraid that if we keep adding anything and everything to `meta-zarhus` then it'll become a mess.
And you still need to use `RDEPENDS` not `DEPENDS`